.github/workflows/releaser.yml

```
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  push:
+    tags:
+      # so a vX.Y.Z-test1 doesn't trigger build
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-pre*'
```

```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+-pre.*'
```

Otherwise you accept only `pr`, `pre` and variation of `preeeeee`

---

thank I fixed that bug in fortio/workflows but when I copied pre workflow files from multicurl I forgot about, nice catch